web/client/src/library/components/editor/EditorInspector.tsx

```diff
@@ -485,18 +485,22 @@ function FormDiffModel({
     on,
     where,
   })
+
+  const getTableDiff = useCallback(
+    function getTableDiff(): void {
```

Comment:
Is there a difference if you use an arrow function here instead? Asking to learn what's best practice.

Reply:
No, we can use `arrow function` here instead. I don't know if there really a best practice regarding `arrow function`. I prefer using them as callbacks (like in this snippet i need to change it to arrow to. be consistent :)). The main difference for  me is that regular function will be hoisted where arrow function needs to be defined before usage.